### DISCUSSION TITLE
Monitor device with snmp

### DISCUSSION THREAD

**Discussion by chkronenberg** [jomjol/AI-on-the-edge-device#3390] [Ideas]
To monitor the watermeter hardware it would be helpful if the device would support snmp. In case of an issue with the storage or the cpu temperature i would get an alert.

**Comment by caco3** [comment]
Most user shave a home automation system which can be used for such monitoring.
We have a great integration for Homeassistant.

Beside of this:
- The temperature is very unreliable, don't use it as an indication!
- The SD card can be an issue, although I never had issues even with writing debug logs. If you disable the logging and image storing then there should be now rite access to the SD card at all.
